ITEM_en.tsv

```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
-ITEM_20150317_003886	Research collected by Priest
-ITEM_20150317_003887	Hind Leg of Dumaro
-ITEM_20150317_003888	A part has a lot of meat among Dumaro's hind leg
-ITEM_20150317_003889	Leather front plate
+ITEM_20150317_003884	A lengthy advertisement noting the Kingdom's honor and pride
+ITEM_20150317_003885	Priest's Research
+ITEM_20150317_003886	Research collected by the Priest
+ITEM_20150317_003887	Dumaro's Hind Leg
+ITEM_20150317_003888	The part of a dumaro that has the most meat
+ITEM_20150317_003889	Leather Front Plate
 ITEM_20150317_003891	Leafy Shoulder
-ITEM_20150317_003892	Leather back panel
-ITEM_20150317_003893	Leather chest protector
-ITEM_20150317_003894	Strong roots
-ITEM_20150317_003896	Kepa teeth
+ITEM_20150317_003892	Leather Back Plate
+ITEM_20150317_003893	Leather Chest Protector
+ITEM_20150317_003894	Strong Roots
+ITEM_20150317_003896	Kepa Teeth
 ITEM_20150317_003898	Yokopus Leather
-ITEM_20150317_003900	Vubbe looters Leather
-ITEM_20150317_003902	Key obtained from Yonajolem. I would hope this is so.
+ITEM_20150317_003900	Vubbe Looter Leather
+ITEM_20150317_003902	Key obtained from Yonazolem. Seems to fit a treasure chest of some sort.
 ITEM_20150317_003905	Scripture
-ITEM_20150317_003906	The scripture, If pilgrims is heading to the Cathedral would have it.
-ITEM_20150317_003907	The reagents of testing soul
-ITEM_20150317_003908	Reagents of testing soul that Graschius passed. The key is to secretly spray without monster's reaction.
-ITEM_20150317_003909	Cold fluids of monster
-ITEM_20150317_003914	A piece of paper swallowed by monster. It looks someone's portrait.
-ITEM_20150317_003916	Relics of soldiers
-ITEM_20150317_003917	Relics of soldiers who died to keep the village until the death.
-ITEM_20150317_003918	Food of mine of town
-ITEM_20150317_003919	Food that Vubbe went to looting in the town of mine. The amount is a little less.
-ITEM_20150317_003921	Part of shield when you need to ask Peltasta's cooperation.
-ITEM_20150317_003923	Divine material of words
-ITEM_20150317_003924	Divine Force Detector Bead
-ITEM_20150317_003926	Product of Cathedral
-ITEM_20150317_003928	Devine material of grace
-ITEM_20150323_003929	Sandwich bread
-ITEM_20150323_003930	Sandwich cheese
-ITEM_20150323_003931	Sandwich meat
+ITEM_20150317_003906	The Scripture that anyone calling themselves a Pilgrim of the Cathedral would carry with them.
+ITEM_20150317_003907	Test Soul Reagent
+ITEM_20150317_003908	Reagent for soul testing that Graschius gave you. The key is to secretly spray it without drawing the monster's attention.
+ITEM_20150317_003909	Cold Monster Fluid
+ITEM_20150317_003914	A piece of paper swallowed by a monster. It looks like someone's portrait.
+ITEM_20150317_003916	Soldiers' Relics
+ITEM_20150317_003917	The relics of soldiers who died protecting the Miners' Village.
+ITEM_20150317_003918	Miners' Village Food
+ITEM_20150317_003919	Food that Vubbes plundered in the Miners' Village. Some of it has been eaten.
```

Comment:
*Food that **the* Vubbes plundered in the Miners' Village. Some of it has been eaten.*